```shell
============================= test session starts ==============================
collecting ... collected 2 items

main_test.py::test_pass PASSED                                           [ 50%]
main_test.py::test_failed FAILED                                         [100%]
main_test.py:4 (test_failed)
def test_failed():
>       assert False
E       assert False

main_test.py:6: AssertionError


========================= 1 failed, 1 passed in 0.02s ==========================

Process finished with exit code 1
```